recipes/cadd-scripts/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set version = "1.7.2" %}
-{% set sha256 = "fb2d1deeffe8e1ef5abb042b0bce48f864878b9de80d2a27b16d5a8389e25e20" %}
+{% set sha256 = "bcb62fdd2d77e10d4e39555f7dda1be6a5c64ac89c36f8d598570be951a0a3e1" %}
```

Comment:
Why the SHA256 changes without a change in the version / downloaded file ?

Reply:
Because the version was reassigned to a different commit but the auto bump was already done in bioconda

Reply:
> the version Tag was reassigned to a different commit

Why ?
Why not a new version ?!

Reply:
because!

Reply:
Well, to me this is a bad practice and I won't merge this PR.
Someone else may approve it and merge it.

Versions are cheap. I see no reason to replace the contents of a version. This way the users cannot fall back to a previous version if there is a regression in the new one.

Reply:
Howdy! 

I agree with @martin-g, a minor version update is warranted instead of a retag.

However, we need to get this sorted out. The retagging looks related to: https://github.com/kircherlab/CADD-scripts/issues/78 and https://github.com/kircherlab/CADD-scripts/issues/77

Over on Anaconda, this specific release has only been downloaded 56 times. Those are most likely indexing related, because I imagine if they were users they would have run in to the issues above.

At this point, I will merge this so that the correct version is available to users in the future. But please in the future, if multiple weeks have passed, take the time for a minor version update instead of a shadow retag. Whether its a bad practice or not (it is), we should each recognize the issues the practice of retagging causes for scientific reproducibility.

Cheers, 
Robert